annotator/static/app.js

```diff
@@ -12,5 +12,7 @@ $(() => {
         $container: $("#player"),
         videoSrc: window.video.location,
         videoId: window.video.id,
+        videoStart: window.video.start_time,
```

Comment:
Maybe we just ought to refactor `Player` to take in the video object at this point... what do you think @szhu